tests/io/v3/base/test_explore.py

```diff
@@ -146,3 +147,23 @@ def test_search_response(api):
     )
     assert isinstance(response, Response)
     assert RESPONSE_2 == response.json()
+
+
+@responses.activate
+def test_details_lookup(api):
+    '''
+    Test details lookup endpoint
+    '''
+    sample_respose: str = 'field_1,field_2,field_3,field_4'
+    '\nvalue1,value_2,value_3,value_4'
```

Comment:
This is a demo response It will be updated once the CSV functionality is implemented for an endpoint.